v2/internal/pkg/batch/concurrent_worker.go

```diff
@@ -117,10 +133,15 @@ func (o *ConcurrentBatch) Worker(ctx context.Context, collectorSchema v2alpha1.C
 					case v2alpha1.TypeOperatorBundle, v2alpha1.TypeOperatorCatalog, v2alpha1.TypeOperatorRelatedImage:
 						o.CopiedImages.TotalOperatorImages++
 					}
-				case img.Type != v2alpha1.TypeOCPRelease && img.Type != v2alpha1.TypeOCPReleaseContent:
-					_, errArray = handleError(err, errArray, img, collectorSchema.AllImages)
+				case img.Type.IsOperatorCatalog():
+					operators := collectorSchema.CopyImageSchemaMap.OperatorsByImage[img.Origin]
+					bundles := collectorSchema.CopyImageSchemaMap.BundlesByImage[img.Origin]
+					errArray = append(errArray, mirrorErrorSchema{image: img, err: err, operators: operators, bundles: bundles})
+					spinner.Abort(false)
+				case img.Type.IsAdditionalImage():
+					errArray = append(errArray, mirrorErrorSchema{image: img, err: err})
 					spinner.Abort(false)
-				default:
+				case img.Type.IsRelease(): //TODO ALEX ASK SHERINE IF CINCINNATI SHOULD BE INCLUDED HERE
```

Comment:
@aguidirh - why would you want to skip in a release, if release fails in any way we should abort immediately, as this is critical to install openshift

Reply:
Actually this code was already there and it is the code to abort in case of any failure in a release image as you mentioned above.